libsource/build.gradle

```diff
@@ -47,7 +57,26 @@ dependencies {
     testImplementation 'org.robolectric:robolectric:4.5.1'
     testImplementation 'androidx.test:core:1.4.0'
     testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:$coroutine_version"
+
+    ktlint "com.pinterest:ktlint:$ktlint_version"
 }
-repositories {
-    mavenCentral()
+
+task ktlint(type: JavaExec, group: "verification") {
+    description = "Check Kotlin code style."
+    classpath = configurations.ktlint
+    main = "com.pinterest.ktlint.Main"
+    args "src/**/*.kt",
+            // to generate report in checkstyle format prepend following args:
+            "--reporter=plain", "--reporter=html,output=${buildDir}/ktlint.html"
+    // to add a baseline to check against prepend following args:
+    // "--baseline=ktlint-baseline.xml"
+    // see https://github.com/pinterest/ktlint#usage for more
```

Comment:
Needed to remove some uncommented lines?

Reply:
sounds good, they are removed.